typescript/phoenix-sdk/src/types/Seat.ts

```diff
@@ -13,6 +13,7 @@ export type Seat = {
   market: web3.PublicKey
   trader: web3.PublicKey
   approvalStatus: beet.bignum
+  padding: beet.bignum[] /* size: 16 */
```

Comment:
just a nit: everywhere else uses // for one line comments, and i get ocd about that stuff lol

Reply:
This was autogenerated, not too much we can do here :)

Reply:
We appreciate nits in general though